big_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -300,6 +308,9 @@ end_per_testcase(delete_old_users, Config) ->
                 rpc(mim(), ejabberd_auth, try_register, [Username, Domain, Pass])
         end, Users),
     escalus:end_per_testcase(delete_old_users, Config);
+end_per_testcase(check_password_hash, Config) ->
+    set_store_password(scram),
```

Comment:
If possible I suggest to capture the password format from before the `check_password_hash` test case in the `Config` and here, restore the format to what was captured.

Common test allows you to add some values to the `Config` in `init_per_*` functions so that they can be used in the test case or in the `end_per_*` functions.

Reply:
I think that for most of the tests, there would be nothing to capture as the default is being used. In this case would we capture the fact that it was not set or the value of what is the default? Would restoring of this setting mean that in the end_per_testcase we remove the password_format or set it to the default?

Reply:
Good point :) My goal was to get back to the configuration from before the test case :) If before the test case the format is not set, it'd be good to get back to this situation, if possible. This is mainly to have less suprises if other tests start failing from not obvious reason.